src/components/editTestCase/groupCoverage/QdmGroupCoverage.tsx

```diff
@@ -133,21 +142,33 @@ const QdmGroupCoverage = ({
       }, {});
   };
 
+  const getDefinitionCalcResult = (definitionName) => {
+    const popCriteriaResults =
+      calculationResults[Object.keys(calculationResults)[0]][selectedCriteria];
+    let calcResult;
+    Object.keys(popCriteriaResults.statement_results).forEach((cqlLib) => {
+      calcResult ??=
```

Comment:
??= thats an interesting operator, and I can't seem to google its purpose.

Reply:
and if we found calcResult, why are we not returning.
if (calcResult) return;

Reply:
> only evaluates the right operand and assigns to the left if the left operand is [nullish](https://developer.mozilla.org/en-US/docs/Glossary/Nullish) (null or undefined).

See: [MDN Nullish Coalescing Assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)

Reply:
Thats good to know.

Reply:
I think I understood it, Looks like we are getting the calcResult for the definitionName found in the first library and ignoring all the others. 

I can't figure out the whole picture here, 
but Can we trust that the first element/key of `Object.keys(popCriteriaResults.statement_results) `is always the appropriate library that we need ?